src/libs/analytics.test.ts

```diff
@@ -0,0 +1,63 @@
+export {}
+
+jest.mock('@react-native-firebase/analytics')
+
+const { AnalyticsEvent } = jest.requireActual('./analytics')
+
+describe('analytics', () => {
+  describe('AnalyticsEvent', () => {
+    it('actual members should respect Firebase naming rules', () => {
+      for (const member of Object.values(AnalyticsEvent)) {
+        const isValidMember = validateAnalyticsEvent(member as string)
+        if (!isValidMember) {
+          fail(`${member} is not valid`)
+        }
+      }
+    })
+    it('fake member should fail the test when it is too long', () => {
+      expect(validateAnalyticsEvent('a'.repeat(39))).toBeTruthy()
+      expect(validateAnalyticsEvent('a'.repeat(40))).toBeTruthy()
+      expect(validateAnalyticsEvent('a'.repeat(41))).toBeFalsy()
+    })
+    it('fake member should fail the test when its first letter is not alphabetic', () => {
+      expect(validateAnalyticsEvent('!test')).toBeFalsy()
+      expect(validateAnalyticsEvent('1test')).toBeFalsy()
+      expect(validateAnalyticsEvent('_test')).toBeFalsy()
+    })
+    it('fake member should fail the test when it contains a non alpha numeric char', () => {
+      expect(validateAnalyticsEvent('Event-Testé')).toBeFalsy()
+    })
+    it('fake member should fail the test when it includes a "-"', () => {
+      expect(validateAnalyticsEvent('Event-Test')).toBeFalsy()
+    })
+    it('fake member should fail the test when it includes a reserved keyword', () => {
+      expect(validateAnalyticsEvent('firebase_test')).toBeFalsy()
+      expect(validateAnalyticsEvent('google_test')).toBeFalsy()
+      expect(validateAnalyticsEvent('ga_test')).toBeFalsy()
+    })
+  })
+})
+
+const RESERVED_KEYWORDS = ['firebase_', 'google_', 'ga_']
```

Comment:
```suggestion
const RESERVED_PREFIXES = ['firebase_', 'google_', 'ga_']
```

c'est pour ça que j'ai rajouté `expect(validateAnalyticsEvent('test_firebase_test')).toBeTruthy()` comme test